src/classpath.rs

```diff
@@ -12,6 +12,13 @@ pub fn should_use_library(lib: &Library) -> bool {
     return is_all_rules_satisfied(rules);
 }
 
+
+#[cfg(target_os = "linux")]
+pub const CLASSPATH_SEP: char = ':';
+
+#[cfg(not(target_os = "linux"))]
+pub const CLASSPATH_SEP: char = ';';
```

Comment:
Classpath separator for Windows is `;`, for Linux **and** MacOS `:`

You should use :
```rs
#[cfg(not(target_os = "windows"))] // Linux and MacOS
pub const CLASSPATH_SEP: char = ':';

#[cfg(target_os = "windows")] // Windows
pub const CLASSPATH_SEP: char = ';';
```